react-props/my-app/src/App.tsx

```diff
@@ -0,0 +1,31 @@
+import { Header } from './Header';
+import { ImageContainer } from './ImageContainer';
+import { ImageCaption } from './ImageCaption';
+import { ImageDescription } from './ImageDescription';
+import { ButtonContainer } from './ButtonContainer';
+import './App.css';
+
+const headerText = 'React Image Bank';
+const imageSrc = '/starry-sky.jpeg';
+const imageCaptionText = 'A beautiful Image of Space';
+const imageDescriptionText = `Lorem ipsum, dolor sit amet consectetur
+adipisicing elit. Aliquam incidunt sit dolorum accusantium sapiente reiciendis
+maxime dolores ullam delectus, commodi placeat eveniet, quam voluptates facere
+et magnam architecto quaerat velit voluptas rerum quos asperiores quis in!
+Nemo aut aliquam quas dolore quaerat, error numquam odio modi eos vero. Modi
+nisi eos autem? Iusto hic impedit aspernatur labore quos nesciunt? Ad dolore
+pariatur aperiam nemo, maiores laboriosam quam quod nisi ipsum, dicta sunt
+laborum similique repellat?`;
+const buttonContainerText = 'Click For Next Image';
+
+export default function App() {
+  return (
+    <>
+      <Header headerText={headerText} />
+      <ImageContainer imgSrc={imageSrc} />
```

Comment:
I suggest you be consistent in your naming. Here you use `img` but below you use `image`.

For what it's worth, in my opinion, the use of either `img` or `image` is redundant and therefore detracts from the beauty of the code. We know from the component name that it holds an image, so `src` should be good enough to indicate what the prop holds.

Similarly, just `text` should be adequate for the other components. I would, though, use something more descriptive than `text` where it makes sense. For example, `title` for the header, `label` for the button, etc.